App.js

```diff
@@ -32,3 +36,6 @@ const styles = StyleSheet.create({
 
 
 });
+
+
+//why program still running after line 20 is commented , according to Docs navigation props only passed to that component which is covered with NavigationContainer
```

Comment:
Add questions in comment or in the channel. Adding them in the code would make you to create a new commit without the comment once the question is answered.